src/hotspot/share/cds/heapShared.hpp

```diff
@@ -418,9 +421,21 @@ class HeapShared: AllStatic {
 
   // Run-time only
   static void clear_root(int index);
+
+  static void set_runtime_delta(ptrdiff_t offset) {
```

Comment:
For consistency, `offset` should be renamed to `delta`.

Reply:
Done.